src/ImageSharp/Processing/Processors/Quantization/EuclideanPixelMap{TPixel}.cs

```diff
@@ -226,18 +233,20 @@ public bool TryGetValue(Rgba32 rgba, out short match)
         /// Clears the cache resetting each entry to empty.
         /// </summary>
         [MethodImpl(InliningOptions.ShortMethod)]
-        public void Clear() => this.table.GetSpan().Fill(-1);
+        public readonly void Clear() => this.table.GetSpan().Fill(-1);
```

Comment:
I thinks it's not readonly.

Reply:
Interesting... I only changed these because the IDE suggested it. 

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0251

Reply:
Actually, just asked in the Discord server about this and got this reply which was how I thought it worked.

"readonly only concerns itself with whether it modifies this, not if it is transitively readonly"

Reply:
Aya, I've forgotten that Roslyn only sees "this" and doesn't look further.
Thanks for the update!